Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -574,6 +574,36 @@ static RefPtr<CSSValue> positionOffsetValue(const RenderStyle& style, CSSPropert
     return CSSPrimitiveValue::create(CSSValueAuto);
 }
 
+// FIXME: Replace calls to this function with `getCSSPropertyValuesForShorthandProperties`
+// once white-space longhands are completely implemented.
+Ref<CSSValue> ComputedStyleExtractor::getWhiteSpaceValue(const RenderStyle& style)
+{
+    WhiteSpaceCollapse wsc = style.whiteSpaceCollapse();
+    TextWrap tw = style.textWrap();
```

Comment:
nit: no abbreviation, also you can use `auto` here.
```suggestion
    auto whiteSpaceCollapse = style.whiteSpaceCollapse();
    auto textWrap = style.textWrap();
```